torch/csrc/jit/passes/lower_tuples.cpp

```diff
@@ -76,6 +77,41 @@ void removeTupleNodes(Node* n, bool must_remove_tuples) {
 
 static void LowerAllTuples(Block* block);
 
+static void removeTupleConstant(Node * n) {
```

Comment:
I think this needs to be part of LowerAllTuples, because  right now LowerAllTuple doesn't lower all tuples, which is misleading.